modules/pipeline/pipengine/reconciler/pipeline_gc_compensator.go

```diff
@@ -72,7 +72,7 @@ func (r *Reconciler) getNeedGCPipelines(pageNum int, isSnippet bool) ([]spec.Pip
 	}
 	req.AllSources = true
 	req.IncludeSnippet = isSnippet
-	pipelines, _, total, _, err := r.dbClient.PageListPipelines(req)
+	pipelines, _, _, _, err := r.dbClient.PageListPipelines(req)
```

Comment:
PageListPipelines has too many returns, should refactor it

Reply:
yes, but not now